RobotFramework/Configuration/Uat_Env.robot

```diff
@@ -0,0 +1,5 @@
+*** Variables ***
+${URL}          https://uat.goibibo.com/
+${URL}          https://uat.soucedemo.com/
+${BROWSER}      Chrome
+${ENV}          Uat
```

Comment:
ENV variable we can remove from here and parse as commandline argument
Please remove and commit from all env files